kymatio/scattering1d/frontend/torch_frontend.py

```diff
@@ -103,19 +108,18 @@ def scattering(self, x):
         # treat the arguments
         if self.vectorize:
             size_scattering = precompute_size_scattering(
-                self.J, self.Q, max_order=self.max_order, detail=True)
+                self.J, self.Q, self.T, max_order=self.max_order, detail=True)
         else:
             size_scattering = 0
 
-
-        S = scattering1d(x, self.backend.pad, self.backend.unpad, self.backend, self.J, self.psi1_f, self.psi2_f, self.phi_f,\
-                         max_order=self.max_order, average=self.average,
-                       pad_left=self.pad_left, pad_right=self.pad_right,
-                       ind_start=self.ind_start, ind_end=self.ind_end,
-                       oversampling=self.oversampling,
-                       vectorize=self.vectorize,
-                       size_scattering=size_scattering,
-                       out_type=self.out_type)
+        S = scattering1d(x, self.backend.pad, self.backend.unpad, self.backend, self.J, self.log2_T, self.psi1_f, self.psi2_f,
```

Comment:
while we're at editing stuff, maybe make everything kwargs?

Reply:
I'm in favor but that's better its own stylistic PR